### PR TITLE
Jenkins and quassel CVEs

### PR DESCRIPTION
**Pull request by kurtseifried**

### CODE REVIEW

File: 2018/1000xxx/CVE-2018-1000173.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"data_version": "4.0","references": {"reference_data": [{"url": ""}]},"description": {"description_data": [{"lang": "eng","value": "A session fixaction vulnerability exists in Jenkins Google Login Plugin 1.3 and older in GoogleOAuth2SecurityRealm.java that allows unauthorized attackers to impersonate another users if they can control the pre-authentication session."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "1.3 and older"}]},"product_name": "Jenkins Google Login Plugin"}]},"vendor_name": "Jenkins project"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-04-30T19:35:42.124803","DATE_REQUESTED": "2018-04-16T00:00:00","ID": "CVE-2018-1000173","ASSIGNER": "kurt@seifried.org","REQUESTER": "ml@beckweb.net"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "CWE-384"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000173.json]: JSON: u'' does not match u'^(ftp|http)s?://\\S+$'

File: 2018/1000xxx/CVE-2018-1000176.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"data_version": "4.0","references": {"reference_data": [{"url": ""}]},"description": {"description_data": [{"lang": "eng","value": "An exposure of sensitive information vulnerability exists in Jenkins Email Extension Plugin 2.61 and older in src/main/resources/hudson/plugins/emailext/ExtendedEmailPublisher/global.groovy and ExtendedEmailPublisherDescriptor.java that allows attackers with control of a Jenkins administrator\u201a\u00c4\u00f4s web browser (e.g. malicious extension) to retrieve the configured SMTP password"}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "2.61 and older"}]},"product_name": "Jenkins Email Extension Plugin"}]},"vendor_name": "Jenkins project"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-04-30T19:35:42.126457","DATE_REQUESTED": "2018-04-16T00:00:00","ID": "CVE-2018-1000176","ASSIGNER": "kurt@seifried.org","REQUESTER": "ml@beckweb.net"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "CWE-522"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000176.json]: JSON: u'' does not match u'^(ftp|http)s?://\\S+$'

File: 2018/1000xxx/CVE-2018-1000177.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"data_version": "4.0","references": {"reference_data": [{"url": ""}]},"description": {"description_data": [{"lang": "eng","value": "A cross-site scripting vulnerability exists in Jenkins S3 Plugin 0.10.12 and older in src/main/resources/hudson/plugins/s3/S3ArtifactsProjectAction/jobMain.jelly that allows attacker able to control file names of uploaded files to define file names containing JavaScript that would be executed in another user's browser when that other user performs some UI actions."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "0.10.12 and older"}]},"product_name": "Jenkins S3 Publisher Plugin"}]},"vendor_name": "Jenkins project"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-04-30T19:35:42.126938","DATE_REQUESTED": "2018-04-16T00:00:00","ID": "CVE-2018-1000177","ASSIGNER": "kurt@seifried.org","REQUESTER": "ml@beckweb.net"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "CWE-79"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000177.json]: JSON: u'' does not match u'^(ftp|http)s?://\\S+$'

File: 2018/1000xxx/CVE-2018-1000174.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{"data_version": "4.0","references": {"reference_data": [{"url": ""}]},"description": {"description_data": [{"lang": "eng","value": "An open redirect vulnerability exists in Jenkins Google Login Plugin 1.3 and older in GoogleOAuth2SecurityRealm.java that allows attackers to redirect users to an arbitrary URL after successful login."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "1.3 and older"}]},"product_name": "Jenkins Google Login Plugin"}]},"vendor_name": "Jenkins project"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-04-30T19:35:42.125454","DATE_REQUESTED": "2018-04-16T00:00:00","ID": "CVE-2018-1000174","ASSIGNER": "kurt@seifried.org","REQUESTER": "ml@beckweb.net"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "CWE-601"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000174.json]: JSON: u'' does not match u'^(ftp|http)s?://\\S+$'

File: 2018/1000xxx/CVE-2018-1000175.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{"data_version": "4.0","references": {"reference_data": [{"url": ""}]},"description": {"description_data": [{"lang": "eng","value": "A path traversal vulnerability exists in Jenkins HTML Publisher Plugin 1.15 and older in HtmlPublisherTarget.java that allows attackers able to configure the HTML Publisher build step to override arbitrary files on the Jenkins master."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "1.15 and older"}]},"product_name": "Jenkins HTML Publisher Plugin"}]},"vendor_name": "Jenkins project"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2018-04-30T19:35:42.125952","DATE_REQUESTED": "2018-04-16T00:00:00","ID": "CVE-2018-1000175","ASSIGNER": "kurt@seifried.org","REQUESTER": "ml@beckweb.net"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "CWE-23"}]}]}}
```
Comment (cve-team) [inline 2018/1000xxx/CVE-2018-1000175.json]: JSON: u'' does not match u'^(ftp|http)s?://\\S+$'

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

File: 2018/1000xxx/CVE-2018-1000173.json (hunk 1 above)
Comment (daniel-beck) [reply 2018/1000xxx/CVE-2018-1000173.json]: https://jenkins.io/security/advisory/2018-04-16/#SECURITY-442

Also typo: fixaction

File: 2018/1000xxx/CVE-2018-1000174.json (hunk 4 above)
Comment (daniel-beck) [reply 2018/1000xxx/CVE-2018-1000174.json]: https://jenkins.io/security/advisory/2018-04-16/#SECURITY-684

File: 2018/1000xxx/CVE-2018-1000176.json (hunk 2 above)
Comment (daniel-beck) [reply 2018/1000xxx/CVE-2018-1000176.json]: https://jenkins.io/security/advisory/2018-04-16/#SECURITY-729

File: 2018/1000xxx/CVE-2018-1000177.json (hunk 3 above)
Comment (daniel-beck) [reply 2018/1000xxx/CVE-2018-1000177.json]: https://jenkins.io/security/advisory/2018-04-16/#SECURITY-730

File: 2018/1000xxx/CVE-2018-1000175.json (hunk 5 above)
Comment (daniel-beck) [reply 2018/1000xxx/CVE-2018-1000175.json]: https://jenkins.io/security/advisory/2018-04-16/#SECURITY-784

**Comment by cve-team** [comment]
It looks like this has been rolled into pull request 475. Correct?

George

**Comment by kurtseifried** [comment]
merged into branch DWF-2018-05-01-10

**Comment by kurtseifried** [comment]
merged into branch DWF-2018-05-01-10

**Comment by kurtseifried** [comment]
merged into branch DWF-2018-05-01-10

**Comment by kurtseifried** [comment]
merged into branch DWF-2018-05-01-10

**Comment by kurtseifried** [comment]
I can't close this. Can someone @MITRE please close this request? thanks.

**Comment by cve-team** [comment]
Closing at Kurt's request.

**Pull request closed without merging**